marklogic-data-hub-central/ui/src/assets/mock-data/user-context-mock.ts

```diff
@@ -34,7 +34,7 @@ export const userAuthenticated: IUserContextInterface = Object.assign(defaultUse
     error : {
       type: ''
     },
-    pageRoute: '/',
+    pageRoute: '/tiles',
```

Comment:
@ryanjdew @brucean52 @wooldridge I made this change to mock user context, since an authenticated user will have /tiles as landing page route.